### ISSUE TITLE
AVT1 - Composed Modal DAP violations on close button 

### ISSUE THREAD

**Issue by abbeyhrt** [carbon-design-system/carbon#4707]
Found that all stories of the ComposedModal have DAP violations. 

<img width="524" alt="Screen Shot 2019-11-18 at 12 33 59 PM" src="https://user-images.githubusercontent.com/17281178/69079516-c4b92580-09ff-11ea-813a-efcf4afae1b8.png">

I'm fairly certain that moving the `aria-label` that is on the `Close20` icon to the button above it will remove this violation and since the icons are aria-hidden by default, that should be the right fix!
